llama/llama-3_2-11b-vision-instruct/README.md

```diff
@@ -0,0 +1,71 @@
+# Llama 3.2 11B Vision Instruct VLLM Truss
+
+This is a [Truss](https://truss.baseten.co/) for Llama 3.2 11B Vision Instruct with VLLM. Llama 3.2 11B Vision Instruct is a multimodal (text + vision) LLM. This README will walk you through how to deploy this Truss on Baseten to get your own instance of it.
+
+
+## Deployment
+
+First, clone this repository:
+
+```sh
+git clone https://github.com/basetenlabs/truss-examples/
+cd llama/llama-3_2-11b-vision-instruct
+```
+
+Before deployment:
+
+1. Make sure you have a [Baseten account](https://app.baseten.co/signup) and [API key](https://app.baseten.co/settings/account/api_keys).
+2. Install the latest version of Truss: `pip install --upgrade truss`
+3. Apply for access to the Llama 3.2 11B Vision Instruct model on hugging face [here](https://huggingface.co/meta-llama/Llama-3.2-11B-Vision-Instruct).
+4. Retrieve your Hugging Face token from the [settings](https://huggingface.co/settings/tokens).
+5. Set your Hugging Face token as a Baseten secret [here](https://app.baseten.co/settings/secrets) with the key `hf_access_token`. Note that you will *not* be able to successfully deploy this model without doing this.
+
+With `llama-3_2-11b-vision-instruct` as your working directory, you can deploy the model with:
+
+```sh
+truss push --publish --trusted
+```
+
+Paste your Baseten API key if prompted.
+
+For more information, see [Truss documentation](https://truss.baseten.co).
+
+### Notes
+
+Limitations from VLLM allow for a maximum of 1 image as input. You will get a memory error otherwise. You can keep track of the issue [here](https://github.com/vllm-project/vllm/issues/8826).
+
+## Example usage
+
+```sh
+truss predict -d '{"messages": [{"role": "user", "content": "Tell me about yourself"}]}'
```

Comment:
nit but odd that the example here ≠ example below

Reply:
updated the description to be more clear and mention the different input type (text, and images). I think it's valuable to show both a simple text prompt and an image prompt, with either `truss predict` or via REST